src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -184,7 +184,7 @@ public String getResourceURI(IResource resource) {
 
 	@Override
 	public <T extends IResource> T create(T resource) {
-		return create(resource, resource.getNamespace());
+		return create(resource, resource.getNamespaceName());
```

Comment:
@adietish This is an API change that seems significant

Reply:
@Obirah nice!
@jcantrill nicely spotted! 
I'd ack this as change as preferrable to any kind of more compatible change (ex. call this #getNamespaceResource and and keep the existing #getNamespace) since it's consistent with the existing naming schemes (ex. #getProject retuns IProject, not the project name). Even more since we bumped to the next major lately (6.0.0). 
Thoughts?